docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,7 +110,52 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Updating and Retreiving Playlist
+
+To create a playlist from the server side:
+
+1. Create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavorite");
+    ```
+
+You can create playlist with Metadata also like below code.
+
+    ```
+	var playlist = new MediaControlPlaylist("MyFavoriate",
+		new Dictionary<string, MediaControlMetadata>()
+		{
+			{"Song1", new MediaControlMetadata() {Author = "Someone"} },
+			{"Song2", new MediaControlMetadata()}
+		}
+    );
+	```
+
+To retrieve the playlist and metadata information on the client side:
+    ```
```

Comment:
Kindly check the indentation of this code block. Closing of the code block is with ```` whereas it should be ```

Also while checking it in the view, it looks like this:
![image](https://user-images.githubusercontent.com/42992085/60502093-bf6e6f00-9cda-11e9-8e51-a9d6db7d4597.png)

Please fix the same.

Reply:
Done